Fakebook.Profile/Fakebook.Profile.DataAccess/ProfileRepository.cs

```diff
@@ -72,14 +72,14 @@ private static DomainProfile ToDomainProfile(EntityProfile profile)
         /// <exception type="ArgumentException">If the profile, the profile's email, or the profile's names are null,
         /// this will be thrown </exception>
         /// <returns>A representation of the profile as a DB Entity.</returns>
-        private async Task<EntityProfile> ToEntityProfileAsync(DomainProfile profile)
+        private Task<EntityProfile> ToEntityProfileAsync(DomainProfile profile)
         {
             if (profile == null || profile.Email == null || profile.FirstName == null || profile.LastName == null)
             {
                 throw new ArgumentException("Profile is missing required fields.", nameof(profile));
             }
 
-            var entityProfile = await ConvertAsync(profile);
+            var entityProfile = ConvertAsync(profile);
```

Comment:
confusing variable name; it's not the profile, it's the task. probably just return the method call directly

Reply:
Returning the method call directly now.